src/main/java/org/sagebionetworks/web/server/servlet/UserAccountServiceImpl.java

```diff
@@ -101,15 +88,15 @@ public String createUserStep2(String userName, String fName, String lName, Strin
 			accountSetup.setUsername(userName);
 			accountSetup.setPassword(password);
 			accountSetup.setEmailValidationSignedToken(emailValidationSignedToken);
-			Session s = client.createNewAccount(accountSetup);
```

Comment:
Session is going away.  Use LoginResponse (and pass back access token instead of session token)